### PR TITLE
notun keyword ar translation add  korsi .mama!

### PR DESCRIPTION
**Pull request by nnlgsakib**
Mama josss!
notun kisu keyword add koira disi mama!
thik thakle marge koiro mama!

 fb: [NLG Sakib](https://www.facebook.com/profile.php?id=100075707363159)

### CODE REVIEW

**Comment by ahnafyy** [comment]
@nnlgsakib mama tor prettier formatting toh onek kichu change kore fello? aida chara shudhu changes add kori mama?
and tui jodi asholei prettier project e lagaite chas mama taile akta `.prettierrc` file define kor mama so that consistency thakey. Na hoile dekhbi shobar nijer nijer prettier config e different formatting choila ashbo mama.

Secondly mama,
tui readme and playground e o tor notun keywords gola add koira deh mama and parle akta duita test o likh mama to confirm thik moto shob hoitese mama

File: src/cli.js [hunk 1]
```diff
@@ -1,50 +1,67 @@
...
+  "jotokhon porjonto mama": "while", // While loop
+  "thamis mama": "break", // Break statement
+  "tarpor er tah dekh": "continue", // Continue statement
+  "por por mama": "++", // Increment
+  "kome kome mama": "--", // Decrement
+  "thakbe mama": "function", // Function declaration
+  "kor mama": "", // Function call (You need to provide the function name and arguments)
+  "kore fel mama": "delete", // Delete variable
+  "koyta ache mama": ".length", // Length of an array or string
+  "shuru hoilo": "{", // Start of a block
```
Comment (ahnafyy) [inline src/cli.js]: shuru hoilo, shesh hoilo (start and end of blocks) seems like a bit much? ai duita bad dei? baki gula shob legit kintu tor mama

Comment (nnlgsakib) [reply src/cli.js]: ho mama bad diya daw

**Comment by nnlgsakib** [comment]
> @nnlgsakib mama tor prettier formatting toh onek kichu change kore fello? aida chara shudhu changes add kori mama? and tui jodi asholei prettier project e lagaite chas mama taile akta `.prettierrc` file define kor mama so that consistency thakey. Na hoile dekhbi shobar nijer nijer prettier config e different formatting choila ashbo mama.
> 
> Secondly mama, tui readme and playground e o tor notun keywords gola add koira deh mama and parle akta duita test o likh mama to confirm thik moto shob hoitese mama

thik ase mama  .prettierrc e define koira ditasi

**Comment by ahnafyy** [comment]
@nnlgsakib mama things are looking good now mama. kintu mama ame matro akta notun PR merge marsi mama. ektu rebase kor, ba merge kor mama notun changes gular sathey and then i can approve :D

**Comment by nnlgsakib** [comment]
> @nnlgsakib mama things are looking good now mama. kintu mama ame matro akta notun PR merge marsi mama. ektu rebase kor, ba merge kor mama notun changes gular sathey and then i can approve :D

Akhon ghumaite jabo mama 

Sokale dibone.

Are ami vabtasi playground ta react diye kormu

Tomar loge contact korar acta medium share koro taile personally discuss korte partam

**Comment by ahnafyy** [comment]
React use korar atka bhalo reason deh mama, jodo dependency kom thakbe mama toh better for maintenance?
If we need react because the playground's current state management is falling short I can be convinced to use react :D 

p.s. messaged ya on ig!

**Comment by khaled-0** [comment]
mama 
function call e ``kor mama`` seems unnecessary.
Just function call korlei to better hoy imo.

unnecessary ``kor mama`` which does nothing

same goes for any symbols such as `` + - * / { } ``. they being keywords seems wrong.

**Pull request closed without merging**